app/lib/case_details_section.rb

```diff
@@ -1,15 +1,15 @@
-class ApplicantCaseDetailsSection
+class CaseDetailsSection
   class << self
     def all_steps
-      %i[level_of_help tribunal matter_type applicant dependant_details]
+      %i[level_of_help tribunal asylum_support matter_type]
```

Comment:
One more thing, sorry - as a result of a bunch of discussion over the last week, tribunal screen is being merged into the matter type screen in EL-718 (now in progress), so asylum support will need to come _after_ matter type now.